### PR TITLE
Workspace switch

### PR DESCRIPTION
**Pull request by raintygao**
### Description

add workspace switch

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/public/components/workspace_switch/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useEffect } from 'react';
+import { EuiSelect, EuiTitle } from '@elastic/eui';
+import { useCallback } from 'react';
+import { useOpenSearchDashboards } from '../../../../../plugins/opensearch_dashboards_react/public';
+
+export const WorkspaceSwitch = () => {
```
Comment (ruanyl) [inline src/plugins/workspace/public/components/workspace_switch/index.tsx:11]: @raintygao With @SuZhou-Joe 's latest commit, switching workspace would simply be a url navigation, I think the workspace list can be simply a list of links, which navigate to url `?_workspace_id_=<workspace_id>`

**Comment by ruanyl** [comment]
Is this PR similar to this one https://github.com/ruanyl/OpenSearch-Dashboards/pull/9?

**Comment by raintygao** [comment]
> 

Yes, I have realized this. I think the task assigned to me may only need to call navigate instead of any UI component. Will update and sync in scrum meeting.

**Comment by SuZhou-Joe** [comment]
> > 
> 
> Yes, I have realized this. I think the task assigned to me may only need to call navigate instead of any UI component. Will update and sync in scrum meeting.

There is a method called "formatUrlWithWorkspaceId" in workspaces.client. You can make use of that to get the jumping url.

**Comment by raintygao** [comment]
This pull request is based on https://github.com/ruanyl/OpenSearch-Dashboards/pull/9. Waiting for it merged.

**Comment by raintygao** [comment]
Used a new branch and switched to a new PR.https://github.com/ruanyl/OpenSearch-Dashboards/pull/17 . This PR will be closed.

**Pull request closed without merging**